packages/@aws-cdk/pipelines/README.md

```diff
@@ -144,6 +144,21 @@ new MyPipelineStack(this, 'PipelineStack', {
 });
 ```
 
+Deployment is done by default with `CodePipeline` engine by first generating
+change set, and then executing it. This allow to add e.g. approval before 
```

Comment:
Please proof-read your README section once more before submitting.

```suggestion
a change set, and then executing it. This allows you to add e.g. approval before 
```

----------

The reason we have changeset + execute is not (just) for approval steps. Like I said in a comment to the original issue, certain CloudFormation features require a ChangeSet. Hence, it's only fair to list here what features you will be unable to use if you disable the prepare step.

Reply:
Is there somewhere list of those or documentation about features that require change set? Dynamic referring is only one mentioned in issue.

Reply:
The list has not been added to documentation.
Below in this review, @rix0rrr, you say:

>  I'm told I'm wrong and direct deployments should just work, so the only reason to have changesets might just be to add an approval step.

Is this confirmed that this is the only difference?
(we are using a 30 stacks pipeline, without need for approval step, and saving 1 minute per stack when they are not updated is enormous - it seems to work without any side-effect for now)